### PR TITLE
[0.3] Update configuration documentation for 0.3 features

### PR DESCRIPTION
**Pull request by adamwathan**

### CODE REVIEW

File: docs/source/docs/configuration.blade.md [hunk 1]
```diff
@@ -1,42 +1,214 @@
...
+    // Don't include this module at all.
+    appearance: false,
+    // ...
+  },
+
+  // ...
+}
+```
+
+If a module is missing in your configuration file, the default configuration for that module will be used.
```
Comment (reinink) [inline docs/source/docs/configuration.blade.md]: > If a module is missing **from** your configuration file,

Maybe?

**Review by reinink — APPROVED** [review]
Impressive.

File: docs/source/docs/configuration.blade.md (hunk 1 above)
Comment (adamwathan) [reply docs/source/docs/configuration.blade.md]: Yesss english++.

File: defaultConfig.stub.js [hunk 2]
```diff
@@ -870,8 +870,8 @@ module.exports = {
 
   options: {
     prefix: '',
-    separator: '\\:',
     important: false,
+    separator: '\\:',
```
Comment (m1guelpf) [inline defaultConfig.stub.js]: Didn't you just change this to simply `:`?

Comment (adamwathan) [reply defaultConfig.stub.js]: Yep but neither is merged yet :)

**Pull request merged**